karspexet/show/models.py

```diff
@@ -19,6 +19,28 @@ class Show(models.Model):
     def upcoming():
         return Show.objects.filter(date__gte=datetime.date.today())
 
+    @staticmethod
+    def ticket_coverage():
+        return Show.objects.raw("""
+            select show.id,
```

Comment:
I don't mind the raw SQL, but I find it harder to read without CAPITALIZATION, since you don't really get syntax highlighting inside strings.

Reply:
Oh, I'll definitely fix that.

Reply:
@maxnordlund It is done in 455b378